spec/lib/puppet_spec/ssl.rb

```diff
@@ -134,8 +135,9 @@ def self.revoke(serial, crl, ca_key)
 
     # Creates a self-signed root ca, then signs two node certs, revoking one of them.
     # Creates an intermediate CA and one node cert off of it.
+    # Creates a second intermediate CA and one node cert off if it.
```

Comment:
Did you want to add a visual description of the chain?

Reply:
yes!